jlc_kicad_tools/jlc_lib/cpl_fix_rotations.py

```diff
@@ -108,20 +131,23 @@ def FixRotations(input_filename, output_filename, db):
                     row[side_index].strip() == "bottom" and float(row[posx_index]) < 0.0
                 )
                 if flip_x:
-                    row[posx_index] = "{0:.6f}".format(-float(row[posx_index]))
+                    posx = -posx
 
                 row[ref_index] = row[ref_index].upper()
-                last_correction = None
-                for pattern, correction in db.items():
+                last_entry = None
+                last_pattern = None
+
+                for pattern, entry in db.items():
                     if pattern.match(row[package_index]):
-                        _LOGGER.logger.info(
-                            "Footprint {} matched {}. Applying {} deg correction".format(
-                                row[package_index], pattern.pattern, correction
-                            )
-                        )
-                        last_correction = correction
-
-                if last_correction is not None:
+                        last_entry = entry
+                        last_pattern = pattern
+
+                del entry
+                del pattern
+
+                if last_entry is not None:
+                    _LOGGER.logger.info(f"Footprint {row[package_index]} matched {last_pattern.pattern}. Applying {last_entry.rotation} deg rotation correction")
```

Comment:
Log at the end of the if block instead and include both rotation and offset?

Reply:
Sure.